src/assets/stylesheets/containers/experience.sass

```diff
@@ -1,111 +1,131 @@
-.heading
-  font-size: $font-size-xl * 2
+.card
+  background-color: $color-white
+  margin-top: $m-lg
 
-  &--primary-branded
-    @extend .heading
-    &::before
-      @extend .divider
-      content: ' '
-      display: block
-      margin: 0 auto $m-xl auto
+  &__caption
+    font:
+      size: $font-size-xl
+      weight: $font-weight-bold
+    margin:
+      block-start: 0
+      block-end: 0
+    padding:
+      bottom: $m-sm
+      left: $m-sm
 
-  &--secondary-branded
-    @extend .heading
-    &::before
-      @extend .divider
-      content: ' '
-      display: block
-      margin:
-        top: $m-xl
-        bottom: $m-xl
+  &__image
+    height: $m-xxl * 4
+    overflow: hidden
+
+  &__subcaption
+    border-bottom: $m-xs/4 solid $color-gray
+    color: $color-gray
+    display: inline-block
+    font-size: $font-size-sm
+    margin: $m-sm
+    padding: $m-sm
+    text-transform: uppercase
 
 .column-section
-  margin:
-    left: $m-xxl * 2
-    right: $m-xxl * 2
   display: flex
-  justify-content: space-around
-  align-items: flex-start
+  flex-direction: column
 
   &__column
-    flex: 1 1 33%
     display: flex
-    justify-content: center
     flex-direction: column
-    &:not(:last-child)
-      margin-right: $m-lg
+    flex: 1 1 33%
+    justify-content: center
 
   &__description
-    font-size: $font-size-lg * 1.1
+    border-bottom: 1px solid rgba($color-gray, 0.3)
     color: rgba($color-gray, 0.5)
-    padding:
-      bottom: $m-xl
+    font-size: $font-size-xl
     margin:
-      top: $m-md
       bottom: $m-xl
-    border:
-      bottom: $m-xs/8 solid rgba($color-gray, 0.3)
-
-.card
-  background-color: $color-white
-  &--visible-on-top
-    @extend .card
-    position: relative
-    z-index: $z-higher
-    box-shadow: -$m-lg $m-lg $m-xl rgba($color-violet, 0.2)
+      top: $m-md
+    padding-bottom: $m-xl
 
-  &:last-child:nth-child(n+2)
-    margin-top: $m-xl
-    position: relative
-    left: 55px
+.divider
+  background-color: $brand-primary
+  border-radius: $border-radius-sm
+  border: $m-xs/8 solid $brand-primary
+  height: $m-xs
+  width: $m-xxl * 1.5
```

Comment:
This should be either:
* part of `.heading`
* a mixin that you will import

To avoid @extend-ing